xml/System.Security.Cryptography.Pkcs/SubjectIdentifier.xml

```diff
@@ -52,10 +52,19 @@
         <Parameter Name="certificate" Type="System.Security.Cryptography.X509Certificates.X509Certificate2" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="certificate">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="certificate">The certificate to verify for a match.</param>
+        <summary>Verifies if the specified certificate was used for the decryption operation.</summary>
+        <returns><see langword="true" /> if the specified certificate was used for decryption; otherwise, <see langword="false" />.</returns>
```

Comment:
I'd go with something more like:

`true if the specified certificate's SKI matches current subject identifier instance` (SKI=Subject Key Identifier if we prefer expanded names)

Reply:
(I'm not sure this instance can only be obtained by decryption APIs so it's safer to write it in more generic way and will also be more future proof)

Reply:
```suggestion
        <returns><see langword="true" /> if the specified certificate's identifier matches the current subject identifier instance; otherwise, <see langword="false" />.</returns>
```

Reply:
It's sort of weirder than that.  It's like "returns true if the certificate has characteristics that match the subject identifier"; there's not necessarily "an identifier" that the certificate "has".  (In the case of Issuer+SN it's two different values that have to both match, in the case of SKI it's either an explicit or an implicit value)